config/src/test/java/org/springframework/security/config/annotation/web/configuration/OAuth2ClientConfigurationTests.java

```diff
@@ -107,6 +110,11 @@ public ClientRegistrationRepository clientRegistrationRepository() {
 		public OAuth2AuthorizedClientRepository authorizedClientRepository() {
 			return AUTHORIZED_CLIENT_REPOSITORY;
 		}
+
+		@Bean
+		public OAuth2AccessTokenResponseClient<OAuth2ClientCredentialsGrantRequest> accessTokenResponseClient() {
```

Comment:
>  I wasn't sure what the best way to test this was...

We can test to see if `OAuth2AccessTokenResponseClient<OAuth2ClientCredentialsGrantRequest>` was actually used. We can `mock(OAuth2AccessTokenResponseClient.class)` and test it's use using `verify(mockAccessTokenResponseClient).getTokenResponse(...`. Makes sense?